Structure_oM/Loads/GeometricalLineLoad.cs

```diff
@@ -62,6 +63,9 @@ public class GeometricalLineLoad : BHoMObject, ILoad
         [Description("Line defining the location of the load.")]
         public virtual Line Location { get; set; } = null;
 
+        [Description("A collection of IAreaElements (e.g. Panels) to apply the line load to. These are not required for all adapters.")]
+        public virtual BHoMGroup<IAreaElement> Objects { get; set; } = null;
```

Comment:
```suggestion
        public virtual BHoMGroup<IAreaElement> Objects { get; set; } = new BHoMGroup<IAreaElement>();
```

If you give a `null` default value, you will cause an error here:
https://github.com/BHoM/Robot_Toolkit/blob/aafb6067b31db673ca3a08c0ea7e08bb07ee2b3e/Robot_Adapter/CRUD/Create/Loads/Loads.cs#L97-L99